packages/cice5/package.py

```diff
@@ -25,45 +25,25 @@ class Cice5(MakefilePackage):
     depends_on("mpi")
     depends_on("netcdf-fortran@4.5.2:")
     depends_on("netcdf-c@4.7.4:")
-    # TODO: For initial verification we are going to use static pio.
-    #       Eventually we plan to move to shared pio
-    # ~shared requires: https://github.com/spack/spack/pull/34837
-    depends_on("parallelio~pnetcdf~timing~shared")
     depends_on("datetime-fortran")
     depends_on("oasis3-mct+deterministic", when="+deterministic")
     depends_on("oasis3-mct~deterministic", when="~deterministic")
-    depends_on("libaccessom2+deterministic", when="+deterministic")
-    depends_on("libaccessom2~deterministic", when="~deterministic")
 
-    phases = ["edit", "build", "install"]
+    # Copied with when() from MOM5 SPR
```

Comment:
```suggestion
```

Reply:
That comment is important because it tracks the origin of a non-obvious conditional.

Reply:
I don't think the origin is important?

A comment saying "use master version dependencies as default" but with better wording than mine could be more useful

Reply:
The origin is important because that how it can be tracked back to the origin git commit and PR.

Reply:
The origin is important because that's how it can be tracked back to the original git commit and PR.

Reply:
I'm adding `These are the default dependencies when building version "master".`